components/build/tekton-chains/kustomization.yaml

```diff
@@ -7,6 +7,9 @@ resources:
 #  curl -s https://storage.googleapis.com/tekton-releases/ | xq | grep -E 'chains/.*/release.yaml'
 #
 - https://storage.googleapis.com/tekton-releases/chains/previous/v0.8.0/release.yaml
+- chains-certs-secret.yaml
+- chains-certs-rbac.yaml
+- chains-certs-job.yaml
```

Comment:
Would it be tidier/better if these were concatenated in one big yaml?

Reply:
Yes, changed it.